x-pack/plugins/cases/public/components/case_view/case_view_page.tsx

```diff
@@ -6,131 +6,49 @@
 import { CaseViewMetrics } from './metrics';
-import type { CaseViewPageProps, OnUpdateFields } from './types';
+import type { CaseViewPageProps } from './types';
 import { useCasesFeatures } from '../cases_context/use_cases_features';
-
-const useOnUpdateField = ({
-  caseData,
-  caseId,
-  handleUpdateField,
-}: {
-  caseData: Case;
-  caseId: string;
-  handleUpdateField: (newCase: Case, updateKey: UpdateKey) => void;
-}) => {
-  const { isLoading, updateKey: loadingKey, updateCaseProperty } = useUpdateCase({ caseId });
-
-  const onUpdateField = useCallback(
-    ({ key, value, onSuccess, onError }: OnUpdateFields) => {
-      const callUpdate = (updateKey: UpdateKey, updateValue: UpdateByKey['updateValue']) =>
-        updateCaseProperty({
-          updateKey,
-          updateValue,
-          updateCase: (newCase) => handleUpdateField(newCase, updateKey),
-          caseData,
-          onSuccess,
-          onError,
-        });
-
-      switch (key) {
-        case 'title':
-          const titleUpdate = getTypedPayload<string>(value);
-          if (titleUpdate.length > 0) {
-            callUpdate('title', titleUpdate);
-          }
-          break;
-        case 'connector':
-          const connector = getTypedPayload<CaseConnector>(value);
-          if (connector != null) {
-            callUpdate('connector', connector);
-          }
-          break;
-        case 'description':
-          const descriptionUpdate = getTypedPayload<string>(value);
-          if (descriptionUpdate.length > 0) {
-            callUpdate('description', descriptionUpdate);
-          }
-          break;
-        case 'tags':
-          const tagsUpdate = getTypedPayload<string[]>(value);
-          callUpdate('tags', tagsUpdate);
-          break;
-        case 'status':
-          const statusUpdate = getTypedPayload<CaseStatuses>(value);
-          if (caseData.status !== value) {
-            callUpdate('status', statusUpdate);
-          }
-          break;
-        case 'settings':
-          const settingsUpdate = getTypedPayload<CaseAttributes['settings']>(value);
-          if (caseData.settings !== value) {
-            callUpdate('settings', settingsUpdate);
-          }
-          break;
-        default:
-          return null;
-      }
-    },
-    [updateCaseProperty, handleUpdateField, caseData]
-  );
-  return { onUpdateField, isLoading, loadingKey };
-};
+import { useOnUpdateField } from './use_on_update_field';
+import { CaseViewActivity } from './components/case_view_activity';
 
 export const CaseViewPage = React.memo<CaseViewPageProps>(
   ({
     caseData,
     caseId,
     fetchCase,
     onComponentInitialized,
-    actionsNavigation,
+    updateCase,
+    refreshRef,
     ruleDetailsNavigation,
+    actionsNavigation,
     showAlertDetails,
-    updateCase,
     useFetchAlertData,
-    refreshRef,
   }) => {
     const { userCanCrud } = useCasesContext();
     const { metricsFeatures } = useCasesFeatures();
```

Comment:
I think that they are not used in the component

Reply:
they are. All variables and hooks left are used. If not typescript would fail to pass.